### PR TITLE
Clean up profiles

### PR DESCRIPTION
**Pull request by Tomin1**
There is nothing useful in jolla-calculator.profile and jolla-calendar.profile.

Removing private-bin definitions allows to use any binary but at the same time it makes it possible to clean up some more profiles.

Remove Notes.permission.

### CODE REVIEW

File: permissions/jolla-camera.profile [hunk 1]
```diff
@@ -1,16 +0,0 @@
...
-# x-sailjail-translation-key-description =
-# x-sailjail-description = Execute jolla-camera application
-# x-sailjail-translation-key-long-description =
-# x-sailjail-long-description =
-
-### PERMISSIONS
-# x-sailjail-permission = Camera,Pictures,Videos,MediaIndexing,Location,Privileged
-
-### APPLICATION
-private-bin jolla-camera,jolla-camera-lockscreen
```
Comment (rainemak) [inline permissions/jolla-camera.profile:15]: Does jolla-camera-lockscreen work now?

Comment (Tomin1) [reply permissions/jolla-camera.profile:15]: Yes. These private-bin things are required if there is even one of them defined but when there is none, all binaries are available in the sandbox.

File: permissions/jolla-mediaplayer.profile [hunk 2]
```diff
@@ -1,24 +0,0 @@
...
-# x-sailjail-permission = Tracker
-# x-sailjail-permission = Bluetooth
-# x-sailjail-permission = Videos
-# x-sailjail-permission = Connman
-
-### APPLICATION
-private-bin jolla-mediaplayer
-private-etc machine-id
-dbus-user.own com.jolla.mediaplayer
-dbus-user.own org.mpris.MediaPlayer2.jolla-mediaplayer
```
Comment (rainemak) [inline permissions/jolla-mediaplayer.profile:24]: Mpris working now via Audio permission?

File: permissions/jolla-mediaplayer.profile [hunk 3]
```diff
@@ -1,24 +0,0 @@
...
-# x-sailjail-permission = Audio
-# x-sailjail-permission = Notifications
-# x-sailjail-permission = Tracker
-# x-sailjail-permission = Bluetooth
-# x-sailjail-permission = Videos
-# x-sailjail-permission = Connman
-
-### APPLICATION
-private-bin jolla-mediaplayer
-private-etc machine-id
```
Comment (rainemak) [inline permissions/jolla-mediaplayer.profile:22]: Hmm. Why this was here?

Comment (Tomin1) [reply permissions/jolla-mediaplayer.profile:22]: :shrug:

File: permissions/jolla-mediaplayer.profile (hunk 2 above)
Comment (Tomin1) [reply permissions/jolla-mediaplayer.profile:24]: Yes.

**Review by rainemak — APPROVED** [review]
Thanks Tomi!

**Pull request merged**